cmd/read/read.go

```diff
@@ -58,45 +64,50 @@ func (o *Options) Run(cmd *cobra.Command, args []string, cloud *common.Cloud) er
 		return err
 	}
 
-	var events []string
-	for _, event := range tsk.Events(ctx) {
-		events = append(events, fmt.Sprintf(
-			"%s: %s\n%s",
-			event.Time.Format("2006-01-02 15:04:05"),
-			event.Code,
-			strings.Join(event.Description, "\n"),
-		))
-	}
-	logrus.Info(events)
+	switch {
```

Comment:
Shouldn't this be just three conditions instead of a `switch`? Go's `switch` is not _fallthrough_ by default.

Reply:
I guess it doesn't make any difference for this use case, because `o.Status`, `o.Events` and `o.Logs` are mutually exclusive. 🤷🏼‍♂️ 

Although separate conditions are probably cleaner, or even `switch` and separate functions for each `case`; cramming all the code there is probably a bit too much.

Reply:
Having three separate flags define something that's exclusive is counter-intuitive. A nice principle to follow here is to not make it possible to specify a flag combination that is invalid (i.e. `-status -events`). I'd probably go with a `-show [events|logs|status]` flag then.

Reply:
Using [`MarkFlagsMutuallyExclusive`](https://pkg.go.dev/github.com/spf13/cobra#Command.MarkFlagsMutuallyExclusive) doesn't seem precisely counter-intuitive to my mind. See for instance `gh pr merge [--merge|--rebase|--squash]` where `only one of --merge, --rebase, or --squash can be enabled` likewise.

Reply:
Although, admittedly, options on `gh pr merge` represent verbs and options on `task read` represent nouns. 🤔

Reply:
I missed [MarkFlagsMutuallyExclusive](https://pkg.go.dev/github.com/spf13/cobra#Command.MarkFlagsMutuallyExclusive) - I think it's fine as it is

Reply:
Preoposed solution on https://github.com/iterative/terraform-provider-iterative/pull/655/commits/b7722fd70d13b883bbe6d8f900804b6f7a740e3f